libs/agno/agno/tools/user_control_flow.py

```diff
@@ -0,0 +1,61 @@
+from textwrap import dedent
+from typing import Optional
+
+from agno.tools import Toolkit
+
+
+class UserControlFlowTools(Toolkit):
+    def __init__(
+        self,
+        get_user_input: bool = True,
```

Comment:
Don't we always want get_user_input to be True? if not what's the other flow, and should we add cookbook for that?

Reply:
You are right. This is a bad pattern.